package.json

```diff
@@ -169,7 +169,6 @@
     "@uniswap/v3-core": "1.0.0",
     "@uniswap/v3-periphery": "^1.1.1",
     "@uniswap/v3-sdk": "^3.9.0",
-    "@uniswap/widgets": "^2.49.0",
```

Comment:
`@uniswap/widgets` was also the only thing requiring our `resolutions` - can you remove that field from the package.json?